packages/admin/admin-bundler/package.json

```diff
@@ -33,13 +33,14 @@
     "@vitejs/plugin-react": "^4.2.1",
     "autoprefixer": "^10.4.16",
     "compression": "^1.7.4",
+    "express": "^4.21.0",
```

Comment:
I see the usage of express is in context to the Vite Dev middleware and not the Medusa app, so decided to keep it as a regular dependency.

Do let me know, if my assumption is wrong